app/views/administration/orders/_table_row_sending_room.html.erb

```diff
@@ -29,7 +29,7 @@
           <td><%= link_to "DOWNLOAD PDF", administration_order_items_checklist_path(order.id, format: :pdf) %></td>
           <%= form_for order, url: administration_order_path(order.id), html: { method: :patch } do |f| %>
           <td>
-           <%= f.select :status, [["initié",1],["expédié",2]], :selected => order.status, :data_no_turbolink => true , onchange: 'this.form.submit();' %>
+           <%= f.select :status, [["Initiée",0], ["Expédiée",1], ["Livrée",2]], :selected => order.status, :data_no_turbolink => true , onchange: 'this.form.submit();' %>
```

Comment:
Admin can only ship the package . The status delivered is retro-sent from customer or postal service to confirme the delivery

Reply:
Lign modify with your comment